box_embeddings/common/tf_utils.py

```diff
@@ -139,7 +139,7 @@ def softplus_inverse(
         / beta
     )
 
-    return res
+    return tf.constant(res_n)
```

Comment:
Why are we returning a tf constant? Why are we using numpy?

Reply:
Should we return tf.variable?
Since I was returning tf.constant, I couldn't modify its value unless I edit its content in NumPy and return a new constant tensor